apis/v1alpha1/validation.go

```diff
@@ -21,6 +21,15 @@ import (
 	"strings"
 )
 
+var (
+	defaultCLI              = "gcloud"
+	invalidCommandOperators = map[rune]struct{}{
```

Comment:
Why use rune instead of string here?

Reply:
rune is char in go? wider range if using rune/char. for example I only need to check '&' for both & and &&. Also here we are iterating rune ```for _, ch := range c {```